ckanext/dsaudit/actions.py

```diff
@@ -67,7 +75,7 @@ def datastore_create(original_action, context, data_dict):
 def datastore_upsert(original_action, context, data_dict):
     rval = original_action(context, data_dict)
     res = context['model'].Resource.get(rval['resource_id'])
-    if res.url_type not in h.datastore_rw_resource_url_types():
+    if res.url_type not in h.datastore_rw_resource_url_types() or _is_system_user(context):
```

Comment:
Ok, I can see why we might want to hide these (e.g. a data migration)

Reply:
@wardi so we will do the `or _is_system_user` for the delete and upsert? And do the `and _is_system_user` for the Create??

Reply:
I think so, but please check my logic :-)
1. we want create audit if it's either not a sysadmin or it's a datastore rw url_type. So *and* on the short circuit exit
2. we want delete and upsert audit if it's a datastore rw url_type but not for sysadmin actions. So *or* on the short circuit exit

I'd be fine with removing the sysadmin check on delete and upsert. We only keep a dozen records from each call so the history might still be manageable even with automated data migration tasks mixed in

Reply:
removed contextual user checks for the datastore delete and upsert actions.